index.html

```diff
@@ -177,35 +204,40 @@ <h2 class="placard-title">Skills</h2>
     </ul>
     </div>
 </section>
+<div class="background-images-footer">
+<div class="footer-desktop">
```

Comment:
- [ ] Great job. Your app looks really great. Please ensure that the cards are also responsive when switching to the desktop version. They should expand a little den change to the desktop version not staying the same, and a space is instead created beside them. Please look into that. 
your version
![Screenshot (395)_LI](https://user-images.githubusercontent.com/65298048/188014597-3824df4c-a61a-4096-9cd1-29d94ba6a66f.jpg)
figma version when zoomed ( the cards increase as well)

![Screenshot (397)](https://user-images.githubusercontent.com/65298048/188014995-33a45234-4696-4896-b083-e22756dd987f.png)